docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,125 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configuring high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+
+Grafana Mimir can deduplicate data from high-availability (HA) Prometheus setup. In this guide, you will see how to configure
+the deduplication for Grafana Mimir helm deployment using external Consul.
+
+## Before you begin
+
+You need to have a Grafana Mimir installed.
+
+Refer to [Configuring High Availability]({{< relref "../configuring-high-availability-deduplication.md" >}}) documents
+for high level description on the concept how Mimir deduplicate incoming HA samples.
+
+You also have to configure HA for Prometheus or Grafana Agent. Last, you need a KV store. In this guide you will
+use Consul. You will be guided on the setup if you haven't had one.
+
+## Configure Prometheus or Grafana Agent to send HA external labels
```

Comment:
This is covered by the other guide. Should we repeat the info here too?

Reply:
I am worried that keep referring to the external document will make the reader keep jumping around to different documents. External label configuration is the most important part of HA configuration, hence I feel for good reading/guide continuity, it will be better to keep this here.

Reply:
I decided to put a note that this section contains similar content from another HA doc. Please review and advise if this note is useful or better just to remove it.